src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -1454,6 +1451,31 @@ bool PhaseIdealLoop::skeleton_predicate_has_opaque(IfNode* iff) {
   return init != 0;
 }
 
+void PhaseIdealLoop::skeleton_predicate_opaque_helper(Node* n, uint& init, uint& stride) {
```

Comment:
I suggest to rename this method to make it more clear what its purpose is. Maybe `count_opaque_loop_nodes`?